homeassistant/components/threshold/config_flow.py

```diff
@@ -30,13 +30,19 @@ def _validate_mode(data: Any) -> Any:
         vol.Required(
             CONF_HYSTERESIS, default=DEFAULT_HYSTERESIS
         ): selector.NumberSelector(
-            selector.NumberSelectorConfig(mode=selector.NumberSelectorMode.BOX),
+            selector.NumberSelectorConfig(
+                mode=selector.NumberSelectorMode.BOX, step=1e-3
```

Comment:
What impact does step have when it's in box mode?

Reply:
Discussed on Discord. This is ok for now but we'll add a new `any` value for `step` in the next release.